### ISSUE TITLE
Carrier registeration not re-established when mobile signal lost

### ISSUE THREAD

**Issue by TellingBerto** [HelloVolla/ubuntu-touch-beta-tests#55]
Hello,

to whom it may concern.

Steps to verify the issue:

1.   Boot the device.
2.    Enter in SIM pin.
    -> Carrier registration takes place successfully.
3.    Enter in device pin.
4.    Open network settings via Network icon in indicator bar.
    -> Verify on the same screen the name of the carrier is displayed before the triangle.
5.    Enter a room/area without mobile signal.
    -> Carrier name disappeared, triangle is inactive with + sign  (No signal).
6.   Enter a room/area with mobile signal.
    -> "Searching" displayed, triangle remains inactive.

**Result:**
Carrier registration lost.

Version: Ubutnu 16.04 (2020-W40) 
Last update: 03.10.2020

**Comment by Stevie00911** [comment]
Same behaviour on my Vollaphone.
During a ride with the Munich S-Bahn (Metro) with some tunnels the carrier registration was lost.
Only after some restarts the carrier signal came back. 
Version: Ubutnu 16.04 (2020-10-08) dev

**Comment by NotKit** [comment]
It could be caused by the same bug as #51 (HelloVolla/android_device_volla_yggdrasil@4bdb722b). Please test if it still occurs as of development OTA channel version 70.

**Comment by TellingBerto** [comment]
Version: Ubutnu 16.04 (2020-10-13/2) dev

Looks much better.

Carrier registration is re-established when signal was lost. 

**Please note:** 
1. Indicator for signal strength is not updated after the registration was re-established, means triangle remaining empty. (In case the device is rebooted the triangle is filled out.)
-> Calls can be received and placed.
 
2. Issue #51 is not fixed, could not re-establish registration after flight mode.
-> Carrier name disappeared, triangle is inactive with + sign (No signal).

**Comment by JamiKettunen** [comment]
@TellingBerto Is this "signal strength not updating" just a visual issue? How long did you wait for it to try and update? Since the original issue itself as per the title ("Carrier registeration not re-established when mobile signal lost") is now fixed, I think this issue can be closed :)

**Comment by TellingBerto** [comment]
> @TellingBerto Is this "signal strength not updating" just a visual issue? How long did you wait for it to try and update? Since the original issue itself as per the title ("Carrier registeration not re-established when mobile signal lost") is now fixed, I think this issue can be closed :)

I was waiting for several minutes. And after the reboot the indicator for the signal strength was updated immediately from empty to fully filled out.
I don´t mind to to close the issue. But would you please open a new one, because it is odd when we can´t relay on the indicators which are displayed.

**Comment by wurzer** [comment]
You can open a new ticket, but please check, whether you have network access and it's a display issue.

**Comment by TellingBerto** [comment]
Version: Ubutnu 16.04 (2020-10-14/2) dev

Steps to verify the issue:

    Boot the device.
    Enter in SIM pin.
    -> Carrier registration takes place successfully.
    Enter in device pin.
    Open network settings via Network icon in indicator bar.
    -> Verify on the same screen the name of the carrier is displayed before the triangle.
    Enter a room/area without mobile signal.
    -> Carrier name disappeared, triangle is inactive.
    Enter a room/area with mobile signal.
    -> "Searching" displayed, triangle remains inactive.
    -> No carrier name displayed in network settings.

But: 
Call can be received from another phone.
No calls can be placed form the device.

Finally, after several minutes (>5) the network access could be established by the device, and network indicator was displayed as expected.
